cmd/frontend/graphqlbackend/graphqlbackend.go

```diff
@@ -56,6 +56,7 @@ func (t *prometheusTracer) TraceQuery(ctx context.Context, queryString string, o
 
 	ctx = context.WithValue(ctx, sgtrace.GraphQLQueryKey, queryString)
 
+	// DEPRECATED - LOG_ALL_GRAPHQL_REQUESTS env variable, and the associated handler logic, will be removed in favor of log.auditLog.graphQL site config setting
```

Comment:
We should schedule a specific removal, n+2 versions is 4.2 which seems like ample time:

```suggestion
	// TODO(4.2)
	// DEPRECATED - LOG_ALL_GRAPHQL_REQUESTS env variable, and the associated handler logic, will be removed in favor of log.auditLog.graphQL site config setting
```